scio-bigtable/src/main/scala/com/spotify/scio/bigtable/package.scala

```diff
@@ -177,6 +177,18 @@ package object bigtable {
     }
 
     /**
+      * Get size of all clusters for specified Bigtable instance.
+      *
+      * @return map of cluster name to it's number of nodes
+      */
+    def getBigtableClusterSizes(projectId: String,
+                                instanceId: String): Map[String, Int] = if (!self.isTest) {
```

Comment:
you mean the if , as follow ?
 ....Map[String, Int] = {
   if (!self.isTest) ..
}